index.js

```diff
@@ -3,53 +3,194 @@ const state = {
     {
       id: "001-beetroot",
       name: "beetroot",
-      price: 0.35
+      price: 0.35,
+      type: "vegetable"
     },
     {
       id: "002-carrot",
       name: "carrot",
-      price: 0.35
+      price: 0.35,
+      type: "vegetable"
     },
     {
       id: "003-apple",
       name: "apple",
-      price: 0.35
+      price: 0.35,
+      type: "fruit"
     },
     {
       id: "004-apricot",
       name: "apricot",
-      price: 0.35
+      price: 0.35,
+      type: "fruit"
     },
     {
       id: "005-avocado",
       name: "avocado",
-      price: 0.35
+      price: 0.35,
+      type: "fruit"
     },
     {
       id: "006-bananas",
       name: "bananas",
-      price: 0.35
+      price: 0.35,
+      type: "fruit"
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
-      price: 0.35
+      price: 0.35,
+      type: "fruit"
+      // Yes I checked and it is a fruit
     },
     {
       id: "008-berry",
       name: "berry",
-      price: 0.35
+      price: 0.35,
+      type: "fruit"
     },
     {
       id: "009-blueberry",
       name: "blueberry",
-      price: 0.35
+      price: 0.35,
+      type: "fruit"
     },
     {
       id: "010-eggplant",
       name: "eggplant",
-      price: 0.35
+      price: 0.35,
+      type: "fruit"
+      // Yes I checked and it is a fruit
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+
+const storeList = document.querySelector('.store--item-list');
+const cartList = document.querySelector('.cart--item-list')
+const total = document.querySelector('.total-number')
+const chosenFilter = document.querySelector('#filter-by-type')
+const cartItems = state.cart
+
+function renderStoreList() {
+  const storeItems = state.items
+
+  for (let i = 0; i < storeItems.length; i++) {
+    let item = storeItems[i];
+
+    const storeListItem = document.createElement('li');
+    storeList.append(storeListItem);
+
+    const storeIconContainer = document.createElement('div');
+    storeIconContainer.setAttribute('class', 'store--item-icon');
+    storeListItem.append(storeIconContainer)
+
+    const storeIcon = document.createElement('img');
+    storeIcon.setAttribute('src', `assets/icons/${item.id}.svg`)
+    storeIconContainer.append(storeIcon);
+
+    const storeButton = document.createElement('button');
+    storeButton.innerText = 'Add to cart'
+    storeButton.addEventListener("click", (e) => {
+      e.preventDefault();
+      
+      const notContainItem = !cartItems.includes(item)
+      const doesContainItem = cartItems.includes(item)
+
+      if(notContainItem){
+        cartItems.push(item)
+        const cartItemIndex = cartItems.length - 1
+        cartItems[cartItemIndex].quantity = 1
+        renderCartList()
+        calcTotal()
+
+        console.log('added cart item!')          
+        } else if (doesContainItem){
```

Comment:
Instead of using an else if statement that accepts your variable. Could you not have used else as the you already have a variable in your first if statement that checks if the cart contains an item?